docs/installation.rst

```diff
@@ -577,3 +577,21 @@ openmm.OpenMMException: Error loading CUDA module: CUDA_ERROR_UNSUPPORTED_PTX_VE
   This error likely means that the CUDA version that ``openmm`` was built with is incompatible with the CUDA driver.
   Try re-making the environment while specifying the correct CUDA toolkit version for your hardware and driver.
   See :ref:`installation:mamba_hpc` for more details.
+
+
+Supported Hardware
+------------------
+
+We currently support the following CPU architectures:
```

Comment:
@richardjgowers is this true? I am not to bothered what we list under "can create a conda env" and "we support" I mostly stuck with `x86_64` since that isn't going away, but if an upstream package decided to drop `osx-arm64`, depending on the package, it would be really hard for us to fix that